packages/core/js-sdk/src/store/index.ts

```diff
@@ -941,6 +941,38 @@ export class Store {
         }
       )
     },
+
+    /**
+     * This method updates the customer of a cart.
+     *
+     * @param id - The cart's ID.
+     * @param body - The data to update in the cart.
+     * @param query - Configure the fields to retrieve in the cart.
+     * @param headers - Headers to pass in the request.
+     * @returns The updated cart.
+     *
+     * @example
+     * sdk.store.cart.updateCustomer("cart_123", {})
+     * .then(({ cart }) => {
+     *   console.log(cart)
+     * })
+     */
+    updateCustomer: async (
```

Comment:
nit/suggestion(non-blocking): Wdyt of `transferCustomer`

Reply:
Yes, my bad. Alternatively `setCustomer`. It's super nitpicky so I'll let you decide, but I just feel like `updateCustomer` intuitively tells me that I can update the customer information (not only the ID) on the cart 😄